.storybook/preview.tsx

```diff
@@ -29,3 +29,7 @@ export const decorators = [
     </Provider>
   ),
 ];
+
+export const parameters = {
+  actions: { argTypesRegex: "^on.*" },
```

Comment:
最近の `sb init` で出力されるものはこのような設定みたい

https://github.com/storybookjs/storybook/blob/db0a3a1108964233669bcbcb028168f2b7bdcd17/lib/cli/src/generators/configure.ts#L71

```suggestion
  actions: { argTypesRegex: "^on[A-Z].*" },
```